### PR TITLE
dev-python/notebook: Use system mathjax

### PR DESCRIPTION
**Pull request by flacjacket**
Also, install all python 2 dependencies for tests

### CODE REVIEW

File: dev-python/notebook/notebook-9999.ebuild [hunk 1]
```diff
@@ -46,6 +48,20 @@ DEPEND="${RDEPEND}
 	)
 	"
 
+python_prepare_all() {
+	# disable bundled mathjax
+	sed -i 's/^.*MathJax.*$//' bower.json
+	sed -i 's/mj(/#mj(/' setupbase.py
```
Comment (jlec) [inline dev-python/notebook/notebook-9999.ebuild]: Always add die to external commands.

**Comment by flacjacket** [comment]
@jlec Added `|| die` to the sed and ln calls

**Comment by marbre** [comment]
I assume we could than also remove line https://github.com/gentoo-science/sci/blob/master/dev-python/ipython/ipython-4.0.0-r1.ebuild#L106 from the ipython ebuild?

**Comment by flacjacket** [comment]
Yeah, the mathjax stuff is in notebook, that line should be removed from
ipython, I can add that to this pr

**Comment by flacjacket** [comment]
@marbre Added

**Comment by marbre** [comment]
Thank you for the pull request. Merged.

**Pull request closed without merging**